proxytest_output.txt

```diff
@@ -0,0 +1,4699 @@
+=== RUN   TestSlowService
```

Comment:
I guess this file you can delete from the PR

Reply:
Oops, sorry about that, I must have added it to the commit without noticing it.